ckan/lib/navl/dictization_functions.py

```diff
@@ -37,6 +37,9 @@ def __str__(self):
             return repr(self.error)
         return ''
 
+    def __unicode__(self):
+        return unicode(str(self))
```

Comment:
Because you'd need to do the `if hasattr(self, 'error') and self.error:` as well.